src/task/update-item-validation-process-task.ts

```diff
@@ -0,0 +1,40 @@
+// global
+import { DatabaseTransactionHandler, Member } from 'graasp';
+// local
+import { ItemValidationService } from '../db-service';
+import { BaseValidationTask } from './base-validation-task';
+import { ItemValidationProcess } from '../types';
+
+type InputType = { id: string; enabled: boolean };
+
+export class UpdateItemValidationProcessTask extends BaseValidationTask<ItemValidationProcess> {
+  input: InputType;
+  reviewer: Member;
+
+  get name(): string {
+    return UpdateItemValidationProcessTask.name;
+  }
+
+  constructor(member: Member, validationService: ItemValidationService, input: InputType) {
+    super(member, validationService);
+    this.input = input;
+    this.reviewer = member;
+  }
+
+  async run(handler: DatabaseTransactionHandler): Promise<void> {
+    this.status = 'RUNNING';
+
+    const { id, enabled } = this.input;
+    const value = enabled? 'TRUE' : 'FALSE';
```

Comment:
Would it be possible to factor out in a variable? And actually I think it is better to keep the true boolean value until db-service, and then use this trick? I think it's possible to transform a boolean to be used in the sql.
```const query = sql`SELECT ${sql.json(true)}`;```